cpu/stm32/Makefile.cmsis

```diff
@@ -3,6 +3,8 @@ PKG_NAME=stm32cmsis
 # The package URL depends on the family of the target STM32
 PKG_URL=https://github.com/STMicroelectronics/cmsis_device_$(CPU_FAM)
 
+# v1.1.0
+PKG_VERSION_c0=7e32bf9d8117ee4c8f6a1d138b814fc24bf4c906
```

Comment:
You'll also have to remove the `ErrorStatus` enum as that conflicts with some packages, see `cpu/stm32/include/vendor/patches`

Reply:
I removed the `ErrorStatus` with a local commit in the `build/stm32/cmsis/c0` and then turned that commit into a patch.  It looks like that's how these vendor patches were done.  Let me know if that's not right.